backend/apps/audit/management/commands/replay_events.py

```
@@ -0,0 +1,230 @@
+            if field == instance._meta.pk.name:
+                continue
+            # Make sure field exists on model
+            if hasattr(instance, field):
+                setattr(instance, field, value)
+
+    def _update_m2m_fields(self, instance, m2m_snapshot: dict):
+        for field_name, value in m2m_snapshot.items():
+            if hasattr(instance, field_name) and isinstance(value, list):
+                try:
+                    getattr(instance, field_name).set(value)
+                except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)